ingestor/.chalice/resources.json

```diff
@@ -74,6 +86,31 @@
             },
             "DD_TAGS": {
               "Ref": "DDTags"
+            },
+            "DD_GIT_REPOSITORY_URL": {
+              "Ref": "DDGitRepositoryUrl"
+            }
+          }
+        }
+      }
+    },
+    "StoreCurrentAlerts": {
+      "Type": "AWS::Serverless::Function",
+      "Properties": {
+        "Description": "Stores all v3 current API alerts",
+        "Environment": {
+          "Variables": {
+            "DD_API_KEY": {
+              "Ref": "DDApiKey"
+            },
+            "DD_VERSION": {
+              "Ref": "GitVersion"
+            },
+            "DD_TAGS": {
+              "Ref": "DDTags"
+            },
+            "DD_GIT_REPOSITORY_URL": {
```

Comment:
Can you explain what this is for? To help us find stuff in DD?

Reply:
This will allow datadog to link stack traces to github lines of code, making it possible to click directly from a stack trace to the related lines of code and the commit it was introduced in